app/styles/u.css

```
@@ -169,3 +169,73 @@
   gap: var(--s-3);
   list-style: none;
 }
+
+/* Users Search page */
+
+.u__search__container {
+  display: flex;
+  align-items: center;
+  margin-top: var(--fonts-xxs);
+  margin-bottom: var(--fonts-xxs);
+  -webkit-box-align: center;
+}
+
+.u__search_container_right {
+  margin-left: var(--fonts-md);
+}
+
+.u__search_discord_container {
+  display: flex;
+  align-items: center;
+  -webkit-box-align: center;
+  font-weight: var(--extra-bold);
+}
+
+.u__search_hyperlink {
+  color: inherit;
+  cursor: pointer;
+  outline: transparent solid 2px;
+  outline-offset: 2px;
+  text-decoration: none;
+}
+
+.u__search_discord_avatar_wrapper {
+  position: relative;
+  display: inline-flex;
+  width: var(--fonts-xl);
+  height: var(--fonts-xl);
+  flex-shrink: 0;
+  align-items: center;
+  justify-content: center;
+  border-color: white;
+  border-radius: var(--border-radius-max);
+  background: rgb(84 30 236);
+  -webkit-box-align: center;
+  -webkit-box-pack: center;
+  color: white;
+  font-size: calc(0.8rem);
+  font-weight: var(--semi-bold);
+  text-align: center;
+  text-transform: uppercase;
+  vertical-align: top;
+}
+
+.u__search_discord_avatar {
```

```suggestion
.u-search__discord-avatar {
```

Should use this sort of naming convention instead to match other style files.